src/app/containers/ChartbeatAnalytics/canonical/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CanonicalChartbeatAnalytics should return script snippet 1`] = `null`;
```

Comment:
This snapshot says 'CanonicalChartbeatAnalytics should return script snippet`, but is returning `null`. This should be returning something, surely?

Reply:
yeah, thanks @rhell. had to change the renderer to use a tree instead.